rulebook.tex

```diff
@@ -2121,63 +2145,71 @@ \subsection{Order Schedule}
 production plan. However, each game will use a new randomized order
 schedule.
 
-\subsection{Scheduled Machine Downtime --- Main Track}
+\subsubsection{Scheduled Machine Downtime --- Main Track}
 There will be exactly two occurrences of scheduled machine downtime for each
 game in the main track according to the rules in \refsec{sec:out-of-order}.
 
-\subsection{Tournament Phases}
+\subsubsection{Overview --- Main Track}
 \label{sec:tournament-phases}
-There will be three stages in the main competition, a round-robin
+There will be three stages in the main track, a (optional) round-robin
 phase for all participating teams, playoffs for the best four teams
 from the round-robin phase, and the finals. The best two teams of the
 playoffs play the grand finale to decide which team will become the
-next Logistics League champion, whereas the other two teams compete in
-the small final for the third place.
+winner of the main track in the \ac{RCLL}, whereas the other two teams compete
+in the small final for the third place.
+
+\paragraph{Scoring Scheme}
+The scoring scheme in the round-robin phase and the playoffs is based on a
+ranking score per won games.
+As each team in this phase directly competes with an opposing team, a ranking
+score will be determined as follows.
+In each \emph{game}, the winning team gets 3 ranking points, the
+loosing team gets 0. A team wins if it achieves more points in the game than the
+other team (cf.~\ref{sec:production-phase}).
+In the case of a non-zero draw, see \refsec{sec:overtime} for overtime.
+If, after overtime, a draw is not resolved, both teams get 1 ranking point.
+If both team end with a zero score, each team gets 0 ranking points.
+Points awarded for commentary are not considered in this decision.
 
-\subsubsection{Round-Robin phase}
-The first stage is a group phase and will be played as a round-robin. The teams
-will receive the true points they scored during the competition. The points will
-be accumulated in this phase and the teams will be ranked according to the
-accumulated points in descending order. Depending on this order, only the first
-6 teams are qualified for the Playoffs.
+\paragraph{Ranking}
+The \emph{overall ranking} is determined by the ranking score of teams, highest
```

Comment:
maybe this is more intuitive: "by the sum of the ranking scores of each team in descending order"